### PR TITLE
Ed nykaza/add vca tests.2

### PR DESCRIPTION
**Pull request by ed-nykaza**
@darinkrauss, please review when you get a chance. Thanks!

### CODE REVIEW

File: nightwatch/commands/initializeNewUser.js [hunk 1]
```diff
@@ -4,12 +4,17 @@ exports.command = function (user) {
     var self = this;
     
     self.perform(function () {
-        var fullName = 'nightwatch+' + user.name + '+' +
-            self.globals.timeTestStarted;
+        // todo: code could break if tests are run in parallel and the timeTestStarted
```
Comment (darinkrauss) [inline nightwatch/commands/initializeNewUser.js]: For future reference, a "to do" tag is usually capitalized to standout. So, `// TODO: <blah>`

File: nightwatch/data/globalTestVariables.js [hunk 2]
```diff
@@ -4,44 +4,64 @@ var moment = require('moment');
 
 module.exports = {
     "waitForConditionTimeout": 10000,
+
```
Comment (darinkrauss) [inline nightwatch/data/globalTestVariables.js]: blank line?

File: nightwatch/tests/loginVCAViewPatientData.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+module.exports = {
+    loginVCAViewPatientData : function (browser) {
+
+        var vca = browser.globals.users.verifiedClinician;
```
Comment (darinkrauss) [inline nightwatch/tests/loginVCAViewPatientData.js]: I think this local variable is unnecessary.

File: nightwatch/tests/loginVCAViewPatientData.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+"use strict";
```
Comment (darinkrauss) [inline nightwatch/tests/loginVCAViewPatientData.js]: Some files use `"use strict";` and others user `'use strict';`.  Please standardize on the latter.

File: nightwatch/commands/initializeNewUser.js (hunk 1 above)
Comment (ed-nykaza) [inline nightwatch/commands/initializeNewUser.js]: done

File: nightwatch/data/globalTestVariables.js (hunk 2 above)
Comment (ed-nykaza) [inline nightwatch/data/globalTestVariables.js]: typo...fixed

File: nightwatch/tests/loginVCAViewPatientData.js (hunk 4 above)
Comment (ed-nykaza) [inline nightwatch/tests/loginVCAViewPatientData.js]: done

File: nightwatch/tests/loginVCAViewPatientData.js (hunk 3 above)
Comment (ed-nykaza) [inline nightwatch/tests/loginVCAViewPatientData.js]: agreed...done

File: nightwatch/commands/signUpNewUser.js [hunk 5]
```diff
@@ -1,9 +1,19 @@
 'use strict';
 
-exports.command = function (user) {
+exports.command = function (user, vca) {
+    
+    var inviteURL = process.env.TIDEPOOL_BLIP_LAUNCH_URL + 'signup/?inviteKey='
```
Comment (ed-nykaza) [inline nightwatch/commands/signUpNewUser.js]: done

File: nightwatch/commands/signUpNewUser.js [hunk 6]
```diff
@@ -1,9 +1,19 @@
...
+            + process.env.TIDEPOOL_BLIP_USER_INVITE_KEY;
+    var accountType = 'personal';
+    
+    if (typeof vca !== 'undefined') {
+        inviteURL = inviteURL + '&roles=clinic';
```
Comment (ed-nykaza) [inline nightwatch/commands/signUpNewUser.js]: done

**Comment by ed-nykaza** [comment]
@darinkrauss, I just addressed all of your suggested changes. Please take a look at the changes when you get a chance. The last thing I need to do for this project is to summarize the possible next actions that someone could take to improve/enhance the tests and test coverage.

**Pull request merged**